### PR TITLE
Add flying velocity to settings

### LINKED ISSUE #184: Add flying mode values to settings

### PR DESCRIPTION
**Pull request by LukeSaward1**
Resolves #184

### CODE REVIEW

File: tool/settings.re [hunk 1]
```diff
@@ -99,6 +99,8 @@ struct Settings {
     player_color_red: float,
     player_color_green: float,
     player_color_blue: float,
+    flying_up_down_vel: float,
+    flying_forward_backward_vel: float,
```
Comment (oberien) [inline tool/settings.re:103]: I don't think shortening `velocity` to `vel` saves much typing here, but definitely hinders understanding the code.

File: tool/settings.re [hunk 2]
```diff
@@ -150,6 +152,8 @@ impl Settings {
             player_color_red: get_float("player_color_red", 0.),
             player_color_green: get_float("player_color_green", 0.),
             player_color_blue: get_float("player_color_blue", 0.),
+            flying_up_down_vel: get_float("flying_up_down_vel", 300.),
+            flying_forward_backward_vel: get_float("flying_forward_backward_vel", 600.),
```
Comment (oberien) [inline tool/settings.re:156]: Are those sane defaults? When I tested, I thought 300 up/down felt pretty slow.

**Review by oberien — CHANGES_REQUESTED** [review]

File: tool/settings.re (hunk 2 above)
Comment (LukeSaward1) [reply tool/settings.re:156]: The latter default is the default in UE.

**Review by oberien — APPROVED** [review]

**Comment by oberien** [comment]
Thanks for the PR :tada:

**Pull request merged**